ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -102,6 +102,12 @@
     reason="Test runs ONLY on AWS deployed cluster"
 )
 
+cloud_platform_required = pytest.mark.skipif(
+    config.ENV_DATA['platform'].lower() != 'aws'
+    and config.ENV_DATA['platform'].lower() != 'azure',
+    reason="Test runs ONLY on AWS, AZURE deployed cluster"
+)
```

Comment:
please use constants.CLOUD_PLATFORMS (just add Azure there) and change to:

```python
cloud_platform_required = pytest.mark.skipif(
    config.ENV_DATA['platform'].lower() not in constants.CLOUD_PLATFORMS
    reason="Test runs ONLY on cloud based deployed cluster"
)
```

Reply:
Thanks , Fixed!